### PR TITLE
add new criteria and list it with 3 others under countermeasure

### PR DESCRIPTION
**Pull request by kristovatlas**
https://github.com/OpenBitcoinPrivacyProject/wallet-ratings/issues/46

### CODE REVIEW

File: obpp3.json [hunk 1]
```diff
@@ -949,7 +949,38 @@
                             "numeral": "1",
                             "id": "OBPPV3/CM31",
                             "description": "Avoid performing queries to third party services regarding specific transactions in a manner which reveals the IP address of the user",
-                            "effectiveness": 0
+                            "effectiveness": 0,
+                            "criteria-groups": [
+                                {
+                                    "criteria": [
+                                        {
+                                            "numeral": "a",
```
Comment (kristovatlas) [inline obpp3.json:957]: Include new comment for CR24: "Tor, I2P, and VPNs are valid methods of hiding machine identity."

File: obpp3.json [hunk 2]
```diff
@@ -949,7 +949,38 @@
...
+                                        {
+                                            "numeral": "b",
+                                            "id": "OBPPV3/CR25",
+                                            "description": "Blockchain data is obtained from a local copy of the blockchain",
+                                            "effectiveness": 0
+                                        },
+                                        {
+                                            "numeral": "c",
+                                            "id": "OBPPV3/CR27",
+                                            "description": "Blockchain data is queried via a method that matches a fraction of the blockchain beyond the addresses belonging to the wallet",
```
Comment (kristovatlas) [inline obpp3.json]: Blockchain data is queried via probabilistic filters matching a fraction of Bitcoin addresses with a transaction history beyond the intent of the query.
JSON Comment: Includes filters with a non-zero fraction of false positives, such as prefix or bloom filters.

File: obpp3.json [hunk 3]
```diff
@@ -949,7 +949,38 @@
...
+                                            "effectiveness": 0,
+                                            "comment": "Score for this criterion should be the mean fraction of the blockchain returned in queries, where downloading the entire blockchain is a perfect score"
+                                        },
+                                        {
+                                            "numeral": "d",
+                                            "id": "OBPPV3/CR170",
+                                            "description": "The wallet client queries for blockchain data via a method that matches a fraction of the blockchain beyond the transactions corresponding to the user's wallet",
```
Comment (kristovatlas) [inline obpp3.json]: Blockchain data is queried via probabilistic filters matching a fraction of Bitcoin transactions beyond the intent of the query.

Comment (crwatkins) [reply obpp3.json]: ack

Comment (dcousens) [reply obpp3.json]: ACK

File: obpp3.json (hunk 2 above)
Comment (dcousens) [reply obpp3.json]: ACK

**Comment by kristovatlas** [comment]
This PR now matches what has been ACK'd.

**Pull request merged**